packages/create-vite/template-vue/src/components/HelloWorld.vue

```diff
@@ -27,7 +27,7 @@ const count = ref(0)
   </p>
   <p>
     Install
-    <a href="https://github.com/vuejs/language-tools" target="_blank">Volar</a>
+    <a href="https://github.com/vuejs/language-tools" target="_blank">Vue - Official</a>
     in your IDE for a better DX
```

Comment:
Would you change the sentence to the following like we did in #16225?

> Learn more about IDE Support for Vue in the [Vue Docs Scaling up Guide](https://vuejs.org/guide/scaling-up/tooling.html#ide-support).

Reply:
Done :)